src/main/java/org/isf/patient/service/FileSystemPatientPhotoRepository.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2021 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.patient.service;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.sql.Blob;
+import java.sql.SQLException;
+
+import javax.sql.rowset.serial.SerialBlob;
+
+import org.isf.generaldata.MessageBundle;
+import org.isf.patient.model.Patient;
+import org.isf.patient.model.PatientProfilePhoto;
+import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.model.OHExceptionMessage;
+import org.isf.utils.exception.model.OHSeverityLevel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component("fileSystemPatientPhotoManager")
+public class FileSystemPatientPhotoRepository {
+
+	private static final String KEY_ERROR_TITLE = "angal.patient.patientphoto.error.title";
+
+	private static final String KEY_FILE_NOT_FOUND = "angal.patient.patientphoto.error.filenotfound.msg";
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(FileSystemPatientPhotoRepository.class);
+
+	private static final String IMAGE_FORMAT = ".png";
+
+	public boolean exist(String path, Integer patientId) {
+		File patientIdFolder = new File(path);
+		File f = new File(patientIdFolder, patientId + IMAGE_FORMAT);
+		return (f.exists() && !f.isDirectory());
+	}
+
+	public void loadInPatient(Patient patient, String path) throws OHServiceException {
+		try {
+			PatientProfilePhoto patientProfilePhoto = new PatientProfilePhoto();
+			patient.setPatientProfilePhoto(patientProfilePhoto);
+			patientProfilePhoto.setPatient(patient);
+			if (exist(path, patient.getCode())) {
+				Blob blob = this.load(patient.getCode(), path);
+				int blobLength;
+				blobLength = (int) blob.length();
+				byte[] blobAsBytes = blob.getBytes(1, blobLength);
```

Comment:
why not just: `byte[] blobAsBytes = blob.getBytes(1, (int) blob.length())` instead of 3 lines?

Reply:
...someone said "make it dumb, improve it later"... here it comes 😃
Done, Thanks!